modules/Components/src/client/OpenExperimentInQueryTool.coffee

```diff
@@ -55,16 +60,29 @@ class OpenExperimentInQueryToolController extends Backbone.View
             error: (err) =>
                 # Take Away Generating Progress Mask 
                 @$('.bv_generatingLink').hide()
-                console.log err
+                toast = new ACASToast
+                    type: "error"
+                    title: "Cannot Generate Link"
+                    text: "ACAS encountered an error when trying to generate a link for the experiment in the Query Tool. Please contact your administrator with this error: #{err.responseText}"
+                    position: "top-middle"
+                    duration: 10000
```

Comment:
Made the duration longer to give people a chance to read the longer error message

Reply:
👍